pkg/worker/config.go

```diff
@@ -52,6 +57,7 @@ func LoadConfig() (*plugin.WorkerConfig, error) {
 	viper.BindEnv("cacert", "CA_CERT")
 	viper.BindEnv("clientcert", "CLIENT_CERT")
 	viper.BindEnv("clientkey", "CLIENT_KEY")
+	viper.BindEnv("statusport", "SONOBUOY_STATUS_PORT")
```

Comment:
We should probably start prefixing our env vars with `SONOBUOY` for clarity/consistency. This is something that should be able to be set by the user to avoid potential conflicts so the worker and the plugin need to get that value from their env.